siliconcompiler/core.py

```diff
@@ -598,16 +565,37 @@ def load_target(self, name, **kwargs):
             Loads the 'freepdk45_demo' target with 5 parallel synthesis tasks
         """
 
-        # Record target
-        self.set('option', 'target', name)
+        # Search order "{name}", and "siliconcompiler.targets.{name}"
+        modules = []
+        for module_name in [name, f'siliconcompiler.targets.{name}']:
+            try:
+                module = importlib.import_module(module_name)
+                modules.append(module)
+            except ModuleNotFoundError:
+                pass
+
+        if len(modules) == 0:
+            self.error(f'Could not find target {name}', fatal=True)
+
+        # Check for setup in modules
+        load_function = None
+        for module in modules:
+            load_function = getattr(module, 'setup', None)
+            if load_function:
+                break
```

Comment:
Do you think it's worthwhile to treat `modules` as a list? It seems like there could only be more than one value if both potential module names are found and loaded (e.g. `mypackage.demo` and `siliconcompiler.targets.mypackage.demo`).

That seems unlikely enough that it might be worth storing the loaded module as a single value, and using `break` in the earlier loop which iterates over the module names. Plus, this loop wouldn't let multiple `setup` methods be called even if multiple modules were imported.

Reply:
we would only want to to call the first one, there would be no need to call multiple. The reason for the list is to get around importing the wrong module which doesn't have a setup function and needing the second found module.